src/db/migrations/migration-2020-05-25-11-18-00-convertData.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const readline = require('readline');
+
+let count = 0;
+let lineBreak = '';
+
+const convertData = () => new Promise((resolve, rejected) => {
```

Comment:
`rejected` is a name for `boolean` variable. This function should be called `reject`.
Will `reject` ever been called?